### PR TITLE
STF 1.5.5 release ops

### PR DESCRIPTION
**Pull request by vkmc**
Depends-On: https://github.com/infrawatch/sg-core/pull/141

### CODE REVIEW

**Comment by vkmc** [comment]
recheck

File: deploy/olm-catalog/smart-gateway-operator/Dockerfile.in [hunk 1]
```diff
@@ -13,7 +13,7 @@ LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v0.19.4
 LABEL operators.operatorframework.io.metrics.project_layout=ansible
 LABEL com.redhat.delivery.operator.bundle=true
-LABEL com.redhat.openshift.versions="v4.12-v4.14"
+LABEL com.redhat.openshift.versions="v4.14-v4.16"
```
[the remark this answers is not in the document]
Comment (vkmc) [reply deploy/olm-catalog/smart-gateway-operator/Dockerfile.in:16]: Good catch. Yes, I think we should drop 4.12 from STO. Keeping it doesn't hurt, but we want to move from it.

[the remark this answers is not in the document]
Comment (vkmc) [reply deploy/olm-catalog/smart-gateway-operator/Dockerfile.in:16]: We already dropped upstream CI https://github.com/infrawatch/service-telemetry-operator/blob/master/.zuul.yaml, so I don't see a point of keeping it. I will propose a change for STO.

**Comment by vkmc** [comment]
recheck

**Comment by vkmc** [comment]
recheck

File: build/Dockerfile
Comment (elfiesmelfie) [inline build/Dockerfile:1]: Should we update this base? I just spotted it with SGO, and may have missed with STO

**Comment by vkmc** [comment]
recheck

**Review by csibbitt — APPROVED** [review]

File: build/Dockerfile
Comment (vkmc) [reply build/Dockerfile:1]: It was updated in STO already :) https://github.com/infrawatch/service-telemetry-operator/commit/27cb54a9fc0eb3cf2ba5c5d91d25443a7be0ff97

**Pull request merged**